task/common/machine/storage.go

```diff
@@ -159,3 +163,28 @@ func Delete(ctx context.Context, destination string) error {
 
 	return nil
 }
+
+func progress(interval time.Duration) func() {
+	accounting.GlobalStats().ResetCounters()
+	ci := fs.GetConfig(context.Background())
+	ci.StatsOneLine = true
+
+	ticker := time.NewTicker(interval)
+	done := make(chan bool)
+
+	go func() {
+		for {
+			select {
+			case <-ticker.C:
+				logrus.Info(accounting.GlobalStats().String())
```

Comment:
maybe `Debug` not `Info`?

Reply:
🙅🏼 I'd prefer to have all the user–facing messages in the `INFO` level and keep `DEBUG` for opt–in diagnostics information.